src/data/de/index.ts

```diff
@@ -171,7 +171,6 @@ export const instanceData = {
     },
     profiles: {
       aboutMe: "Über mich",
-      activeSince: "Bei Serlo seit",
```

Comment:
just for reference: changing `en` should be enough. Other files are generated by Crowdin.